cloud-gateway/src/main/java/com/registry/cloudgateway/controller/CloudGatewayController.java

```diff
@@ -0,0 +1,49 @@
+package com.registry.cloudgateway.controller;
+
+import com.registry.cloudgateway.dto.ClientDTO;
+import com.registry.cloudgateway.dto.InvoiceDTO;
+import com.registry.cloudgateway.service.CloudGatewayService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@EnableEurekaClient
+@RestController
+public class CloudGatewayController {
+
+    @Autowired
+    private CloudGatewayService cloudGatewayService;
+
+    @GetMapping("/client/{id}")
```

Comment:
In this uri, the word client should be plural since without the id part, the endpoint should return all clients (a collection)

Reply:
i have a separate endpoint for all the clients, the one bellow

Reply:
What I meant is that this URI should be /clients/{id}